templates/zerver/app/index.html

```diff
@@ -79,6 +79,27 @@
     border-top-color: hsl(0,0%,52%);
     animation: zspinner 1s linear infinite;
     }
+
+    /* Bootstrap 'a' tag styles for styling reload link */
+    a:focus {
+    outline: thin dotted #333;
+    outline: 5px auto -webkit-focus-ring-color;
+    outline-offset: -2px;
+    }
+    a:hover,
+    a:active {
+    outline: 0;
+    }
+    a {
+    color: #0088cc;
+    text-decoration: none;
+    cursor: pointer;
+    }
+    a:hover,
+    a:focus {
+    color: #005580;
+    text-decoration: underline;
+    }
```

Comment:
So, these styles will remain after the app loads. What this means is I think we want to scope these as only applying to that specific `a` tag.

Reply:
So these can all be, e.g. `#reload-lnk a`